src/fish_key_reader.cpp

```diff
@@ -284,13 +284,12 @@ static bool parse_flags(int argc, char **argv, bool *continuous_mode) {
                 break;
             }
             case 'h': {
-                print_help("fish_key_reader", 0);
-                error = true;
-                break;
+                print_help("fish_key_reader", 1);
+                exit(0);
             }
             case 'v': {
-                std::fwprintf(stdout, L"%s\n", get_fish_version());
-                return false;
+                std::fwprintf(stderr, _(L"%ls, version %s\n"), program_name, get_fish_version());
```

Comment:
Probably this should go to `stdout` as before.

Reply:
Thanks for the review. I copied that snippet from [fish_indent.cpp](https://github.com/fish-shell/fish-shell/blob/3.1.2/src/fish_indent.cpp#L537). Should it be changed to stdout as well?

Reply:
yes the version flags should all print to stdout

Reply:
They print to stdout now.